apps/electron/webpack.config.js

```diff
@@ -0,0 +1,9 @@
+const path = require('path');
+
+module.exports = (config) => {
+  config.entry = {
+    ...config.entry,
+    splash: path.resolve(__dirname, 'src/app/api/splash.preload.ts'),
+  };
+  return config;
+};
```

Comment:
map files are not created (or copy), this does not affect the electron app but throws a warning inside electron chrome. Ensure  he not try to load map files.